engine/graphics/src/graphics.cpp

```diff
@@ -453,7 +453,7 @@ namespace dmGraphics
                 case dmGraphics::VertexAttribute::SEMANTIC_TYPE_TEXCOORD:
                 {
                     uint32_t unit = num_texcoords++;
-                    if (unit >= num_textures)
+                    if (unit >= num_textures || !uvs[unit])
```

Comment:
Crash fix 2: we previously tried to index into the uv list even if we didn't have any valid pointers.